apps/theme/layouts/forms/DefaultFormLayout.tsx

```diff
@@ -39,8 +58,55 @@ const DefaultFormLayout: FormLayoutComponent = ({
     // Is the form successfully submitted?
     const [formSuccess, setFormSuccess] = useState(false);
 
+    // State to show fields corresponding to the step.
+    const [currentStep, setCurrentStep] = useState<number>(0);
+
     // All form fields - an array of rows where each row is an array that contain fields.
-    const fields = getFields();
+    // @ts-ignore
+    const fields = getFields(currentStep);
+
+    // Check if the form is a multi step.
+    const isMultiStepForm = formData.steps.length > 1;
+
+    const handleNextStep = () => {
+        setCurrentStep(prevStep => (prevStep += 1));
+    };
+
+    const handlePrevStep = () => {
```

Comment:
We need to handle step back / step fwd controls and state internally, within the app-form-builder package's component. The same thing we're doing for example with `submit`, pulled from `FormLayoutComponent`. So, let's add those controls above.

The idea is to have these FB layouts as straightforward as possible, and be able to provide as much of OOTB functionality as possible. With this approach, if the user wanted to create another layout, they'd need to c/p again the steps state management code.